### PR TITLE
test(@toss/date): correct unit test describe with 2019년 to 2020년

### PR DESCRIPTION
**Pull request by minsoo-web**
## Overview

Correct unit test describe with `2019` to `2020`

## PR Checklist

- [x] I read and included theses actions below

1. I have read the [Contributing Guide](https://github.com/toss/slash/blob/main/.github/CONTRIBUTING.md)
2. I have written documents and tests, if needed.

### CODE REVIEW

**Review by ChanhyukPark-Tech — CHANGES_REQUESTED** [review]
Why don't you change the test description to English? Slash seems to be ultimately aiming to provide all the codebases in English

**Comment by minsoo-web** [comment]
Thank you, @ChanhyukPark-Tech 🙇‍♂️

> Why don't you change the test description to English? 

As you mentioned `seems`, I think there's nothing decided yet with locale description for test code.

I think this should be added to the contribution guide rather than what my PR should do. If the maintainer sets up a convention to write the test code in English, I will write a PR to correct it.

**Comment by ChanhyukPark-Tech** [comment]
you can check future plan in this [comment](https://github.com/toss/slash/pull/36#issuecomment-1275899085)

**Comment by minsoo-web** [comment]
> you can check future plan in this https://github.com/toss/slash/pull/36#issuecomment-1275899085

Thank you for guide.
I will change this kr to en

File: packages/common/date/src/date.spec.ts [hunk 1]
```diff
@@ -191,7 +191,7 @@ describe('getDateDistanceText', () => {
       ).toEqual('1시간 1분 1초');
     });
 
-    it(`'시간' 단위는 보여지지 않는다.`, () => {
+    it("When don`t wants show '시간' unit.", () => {
```
Comment (ChanhyukPark-Tech) [inline packages/common/date/src/date.spec.ts:194]: `slash` use `it` keyword better than `test` keyword for method

`it` and `test` are functionally the same but semantically different.
```
it("should have a button", () => { ... }); // 1
test("should have a button", () => {}); // 2
```
You can read the code above as english sentence. Then which is the more easily readable? I think 1 is the winner.
```suggestion
    it("should not show '시간' unit", () => {
```

semantically in english

1. it should not show '시간' unit
2. it When don`t wants show '시간' unit.

I think 1 is more readibility🥳

ref: https://github.com/toss/slash/pull/37#discussion_r994630666

Comment (minsoo-web) [reply packages/common/date/src/date.spec.ts:194]: Holy Cool man... i will fix `it`

**Review by ChanhyukPark-Tech — APPROVED** [review]
Cool!

**Review by raon0211 — APPROVED** [review]
Wow! This is great.

**Pull request merged**